bom/build/pom.xml

```diff
@@ -0,0 +1,406 @@
+<?xml version="1.0" encoding="utf-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.hibernate.search</groupId>
+        <artifactId>hibernate-search-parent</artifactId>
+        <version>7.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+
+    <artifactId>hibernate-search-build-bom</artifactId>
+    <name>Hibernate Search - Build BOM</name>
+    <description>
+        Provides dependency management for all dependencies required during the build time.
+        Imports the ones from the public BOM.
+    </description>
+    <packaging>pom</packaging>
+
+    <properties>
+        <!-- Do not publish internal modules -->
+        <skipNexusStagingDeployMojo>true</skipNexusStagingDeployMojo>
```

Comment:
Wait, this one is not internal though, it's referenced from public modules so we do need to publish it?